tests/torch/nas/test_elastic_depth.py

```diff
@@ -204,25 +198,28 @@ def test_can_export_model_with_one_skipped_block_resnet18(tmp_path):
     orig_onnx_model_path = tmp_path / "resnet18.onnx"
     onnx_model_without_block_path = tmp_path / "resnet18_with_one_skipped_block.onnx"
 
-    compressed_model, _ = create_compressed_model_and_algo_for_test(model, nncf_config)
+    compressed_model, ctrl = create_compressed_model_and_algo_for_test(model, nncf_config)
     compressed_model.get_tracing_context().set_elastic_blocks(skipped_blocks)
     # export model to onnx
     ctx = compressed_model.get_tracing_context()
-    nas_export(compressed_model, orig_onnx_model_path)
+    ctrl.export_model(str(orig_onnx_model_path))
 
     ctx.elastic_depth = True  # activate mode with elastic depth
     ctx.set_active_skipped_block([0])
-    nas_export(compressed_model, onnx_model_without_block_path)
+    ctrl.export_model(str(onnx_model_without_block_path))
 
     # load onnx graphs
     # pylint:disable=no-member
     load_model_fn = onnx.load_model
     onnx_resnet18_without_one_block = load_model_fn(onnx_model_without_block_path)
     onnx_resnet18_orig = load_model_fn(orig_onnx_model_path)
 
-    # count of node in skipped block  == 5
-    assert len(onnx_resnet18_orig.graph.node) == 69
-    assert len(onnx_resnet18_without_one_block.graph.node) == 67
+    # Temporary variable for number of nodes is required to avoid hang. When assertion happens on calculation of number
+    # of nodes it tries to output the whole expression (onnx node), and sometimes it causes pytest to freeze.
+    num_all_nodes = len(onnx_resnet18_orig.graph.node)
+    num_not_skipped_nodes = len(onnx_resnet18_without_one_block.graph.node)
+    assert num_all_nodes == 65
```

Comment:
Number of nodes has changed, because NASExporter exported in training mode, but default PTExporter uses evaluation one.

Reply:
Thanks for fixing this.